controller/server_controller.py

```diff
@@ -235,13 +230,16 @@ def get_status(pod) -> ServerStatusEnum:
 
     status = get_status(body)
     server_name = labels[config.PARENT_NAME_LABEL_KEY]
+    now = pytz.UTC.localize(datetime.utcnow())
     api = get_api(config.api_version, config.custom_resource_name, config.api_group)
     api.patch(
         namespace=namespace,
         name=server_name,
         body={
             "status": {
                 "state": status.value,
+                "startingSince": now is status if ServerStatusEnum.Starting else None,
```

Comment:
one liner if/else statements usually confuse me a bit but should this be:
```suggestion
                "startingSince": now if status is ServerStatusEnum.Starting else None,
```